pkg/config/config.go

```diff
@@ -320,9 +320,10 @@ type Config struct {
 	} `mapstructure:"graveler"`
 	Gateways struct {
 		S3 struct {
-			DomainNames Strings `mapstructure:"domain_name"`
-			Region      string  `mapstructure:"region"`
-			FallbackURL string  `mapstructure:"fallback_url"`
+			DomainNames           Strings `mapstructure:"domain_name"`
+			Region                string  `mapstructure:"region"`
+			FallbackURL           string  `mapstructure:"fallback_url"`
+			SkipVerifyUnsupported bool    `mapstructure:"skip_verify_unsupported"`
```

Comment:
This name has too many negatives!  I'd prefer `VerifyUnsupported` or `VerifyOperationStrictly`.

Reply:
Will go with VerifyUnsupported and set it to true in defaults

Reply:
Thanks!